packages/config/src/clusterconfig.ts

```diff
@@ -30,7 +30,7 @@ export const CLUSTER_NAME_REGEX = /^[a-z0-9-_]+$/;
 
 // This is the same regex the k8s ecosystem uses for dns-related label values
 // see https://github.com/opstrace/opstrace/issues/710
```

Comment:
I guess this comment is outdated now - should I edit/remove it?

Reply:
maybe edit it! leave the reference to issue 710 there and add another reference explaining why we also have to remove the dash for now.